proto/cosmos/protocolpool/v1/tx.proto

```diff
@@ -60,6 +59,43 @@ message MsgCommunityPoolSpend {
 
 // MsgCommunityPoolSpendResponse defines the response to executing a
 // MsgCommunityPoolSpend message.
-//
-// Since: cosmos-sdk 0.50
-message MsgCommunityPoolSpendResponse {}
\ No newline at end of file
+message MsgCommunityPoolSpendResponse {}
+
+// MsgSubmitBudgetProposal defines budget proposal type.
+message MsgSubmitBudgetProposal {
+  option (cosmos.msg.v1.signer) = "authority";
+
+  // authority is the address that controls the module (defaults to x/gov unless overwritten).
+  string authority = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // recipient_address is the address of the recipient who can claim the budget.
+  string recipient_address = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // total_budget is the total amount allocated for the budget.
+  cosmos.base.v1beta1.Coin total_budget = 3;
+  // start_time is the time when the budget becomes claimable, represented in seconds since the Unix epoch.
+  // If start_time is less than the current block time, proposal will not be accepted.
+  uint64 start_time = 4;
+  // tranches is the number of times the total budget amount is to be distributed.
+  uint64 tranches = 5;
+  // Period is the time interval(number of seconds) at which funds distribution should be performed.
+  // For example, if a period is set to 3600, it represents an action that
+  // should occur every hour (3600 seconds).
+  uint64 period = 6;
```

Comment:
I would prefer the usage of a concrete duration time type: `durationpb.Duration`